src/main/resources/application-console.yml

```diff
@@ -14,10 +14,9 @@ spring:
     gateway:
       routes:
         - id: architect
-          uri: http://${DOCKER_HOST_IP:172.17.0.1}:9083/
+          uri: http://${DOCKER_HOST_IP:172.17.0.1}:8083/
```

Comment:
Hmm, why was this changed?

Reply:
It's not application-bootRun.yml

Reply:
It's a custom profile, for when you would work on contentcloud-gateway in the context of the contentcloud-api. I would expect that you usually would run the rest of the stack with compose.